utils/network.go

```diff
@@ -82,6 +83,23 @@ func (n Network) URL() string {
 	}
 }
 
+// GatewayURL returns URL for write commands
+func (n Network) GatewayURL() string {
+	switch n {
+	case GOERLI:
+		return "https://alpha4.starknet.io/gateway/"
+	case MAINNET:
+		return "https://alpha-mainnet.starknet.io/gateway/"
+	case GOERLI2:
+		return "https://alpha4-2.starknet.io/gateway/"
+	case INTEGRATION:
+		return "https://external.integration.starknet.io/gateway/"
+	default:
+		// Should not happen.
+		panic(ErrUnknownNetwork)
+	}
+}
+
```

Comment:
Let's try to reuse `URL` function.

We can make URL to only return the base url like `https://alpha4.starknet.io` and callee can append `feeder_gateway` or `gateway`.

Reply:
I'm not sure what you mean by reuse the base URL in this package, so I created a new BaseURL() function and used it in both GatewayURL and FeederURL